openmrs-client/src/main/java/org/openmrs/mobile/models/Visit.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+
+package org.openmrs.mobile.models
+
+import com.google.gson.annotations.Expose
+
+class Visit : Resource() {
+
+    override var id: Long? = null
```

Comment:
Just declare the var, don't assign null to it, since it is required.
```suggestion
    override var id: Long
```

Reply:
Actually, in Kotlin, all the variables must be initialized with some value or it throws an error...
Or there is one option to use `lateinit` but it is causing two problems:
1. Lateinit cannot be applied to primitive data types therefore Long id must be initialized to something
2. Lateinit in `stopDateTime` variable is causing following error in VisitDashboardPresnterTest `rx.exceptions.OnErrorNotImplementedException: lateinit property stopDatetime has not been initialized`

Now one thing can be done is that we leave migration of this class to kotlin as of now... Any other suggestion @f4ww4z sir

Reply:
@rishabh-997 Nevermind then, stick with the current code. `override var id: Long? = null`

Reply:
In that case, this particular conversation is resolved I guess